CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Unreleased
+
+*Breaking:*
+
+- Moved colors to a subspec in `Backpack/Color`.
```

Comment:
Side note, we should add dates to the changelog when we release (YYYY-MM-DD)

Reply:
Sure thing :)